memory/memhotplug.py

```diff
@@ -100,14 +100,19 @@ def setUp(self):
         if not memory.check_hotplug():
             self.cancel("UnSupported : memory hotplug not enabled\n")
         smm = SoftwareManager()
+        for package in ['automake', 'make', 'autoconf']:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.cancel('%s is needed for the test to be run' % package)
+        default_url = 'https://fossies.org/linux/privat/old/stress-1.0.5.tar.gz'
+        self.stress_tar_url = self.params.get('stress_tar_url', default=default_url)
```

Comment:
please make this not a  global variable as it is not accessed outside the Setup function

Reply:
done